tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
We want a flat list of all case relationships, so that would be impossible. I did the initial implementation poorly and currently it goes:

* child
* parent
  * grandparent

When it should have been

* child
* parent
* grandparent